### DISCUSSION TITLE
[Feature]: Replace $CI_COMMIT_MESSAGE for Gitlab migration

### DISCUSSION THREAD

**Discussion by ghost** [github/gh-actions-importer#60] [Feature requests] [answered]
### What operating system are you using?

- [X] Linux
- [ ] macOS
- [ ] Windows

### What version of the tool are you using?

gh version 2.20.2 (2022-11-15)
gh actions-importer     github/gh-actions-importer      v1.0
actions-importer/cli    v1.0.14322

### What happened?

When importing Gitlab CI configurations the environment variable `$CI_COMMIT_MESSAGE` is currently not replaced. We could use `${{ github.event.head_commit.message }}` for replacement.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by ethanis** [comment, marked as answer]
Unfortunately, I don't think using `${{ github.event.head_commit.message }}` is a suitable replacement as that field is only available for `push` events. I'll take this back to the team to see if there's a replacement that would work regardless of the event type.

**Reply by EvanCarroll** [reply]
why is it only available on push events?
